docker/docker-compose.yml

```diff
@@ -16,7 +16,7 @@ version: "3.8"
 
 services:
   postgres:
-    image: "postgis/postgis:13-3.0"
+    image: "postgis/postgis:15-3.3"
```

Comment:
This change seems unrelated, and also, it's set to 13 currently because we use 13 in production.

Reply:
Ugh sorry, I added this because I wanted it compatible with my local DB that runs in `terraware-web`, which is using `15-3.3`. Regardless, I didn't mean to commit it, sorry about that!